common/remote/sockfile.cpp

```diff
@@ -4110,12 +4125,11 @@ class CRemoteDiskReadActivity : public CRemoteDiskBaseActivity
             return true;
     }
 public:
-    CRemoteDiskReadActivity(IPropertyTree &config) : prefetchBuffer(nullptr)
+    CRemoteDiskReadActivity(IPropertyTree &config) : PARENT(config), prefetchBuffer(nullptr)
```

Comment:
PARENT here is just a typedef of the base class? if so is this purely stylistic or are we gaining something

Reply:
yes, just a typedef.
It makes life easier when refactoring that PARENT is always just that - the parent of the current class, e.g. for all the overloaded methods that call back up to parent class.
There aren't many use cases here, but generally in code else where you'll see code like:
```
    virtual void someFunc() override
    {
         PARENT::someFunc();
         dootherstuff..
    }
```